components/channel_header/channel_header.js

```diff
@@ -205,6 +207,15 @@ class ChannelHeader extends React.PureComponent {
         }
     };
 
+    showChannelFiles = (e) => {
+        e.preventDefault();
```

Comment:
Is this necessary? Presumably this is just a pure callback and not a link navigation event we need to cancel or a form we shouldn't submit?

Reply:
:+1:

Reply:
Done